terraform/modules/gsp-cluster/main.tf

```diff
@@ -14,16 +14,12 @@ variable "ssh_authorized_key" {
   type = "string"
 }
 
-variable "codecommit_url" {
-  type = "string"
-}
-
 variable "admin_role_arns" {
   type = "list"
 }
 
 module "cluster" {
-  source = "git::https://github.com/alphagov/gsp-typhoon//aws/container-linux/kubernetes?ref=gsp"
+  source = "git::https://github.com/alphagov/gsp-typhoon//aws/container-linux/kubernetes?ref=flux-spike"
```

Comment:
This change needs cancelling after the other PRs are merged (and before this PR is merged).